ui/app/templates/partials/secret-form-show.hbs

```diff
@@ -1,4 +1,14 @@
-{{#if (and isV2 modelForData.destroyed)}}
+{{#if isWriteWithoutRead}}
+  <EmptyState
+    data-test-write-without-read-empty-message
+    @title="You do not have permission to read this secret."
+    @message={{if isV2
+      "A new version of this secret may be written but your associated policies do not allow you to read its current contents."
+      "Your policies permit you to overwrite the value at this path, but do not allow you to read its current one."
```

Comment:
How about a slightly less technical: "Your policies permit you to overwrite this secret, but do not allow you to read this version."

Reply:
Should we differentiate v1 and v2 ? In v1 you're going to overwrite the value, in v2 you'll create a new version.

Reply:
Oh! I don't know why I misread that. Yeah, maybe something like "Your policies permit you to overwrite this secret, but do not allow you to read it."